*** ../build/test/tlb1378.xetex.tlg	2018-09-03 14:03:23.000000000 +0200
--- ../build/test/tlb1378.xetex.log	2018-09-03 14:03:24.000000000 +0200
***************
*** 86,91 ****
--- 86,93 ----
  ....\hbox(6.44444+0.0)x17.82494, glue set 12.07497fil
  .....\glue 0.0 plus 1.0fil minus 1.0fil
  .....\OT1/cmr/bx/n/10 1
+ .....\kern -1.0
+ .....\kern 1.0
  ....\penalty 10000
  ....\glue(\parfillskip) -15.50005
  ....\glue(\rightskip) 15.50005
